pages/api/movies/[id]/index.js

```diff
@@ -1,12 +1,20 @@
+import dbConnect from "@/components/db/connect";
+import { getMovieInfo } from "@/components/db/utils";
```

Comment:
maybe you can move the db folder out of the components

Reply:
ohh yes, that's right, I just changed it